internal/sirius/document.go

```diff
@@ -26,10 +26,15 @@ const (
 	TypeSave    string = "Save"
 )
 
-func (c *Client) Documents(ctx Context, caseType CaseType, caseId int) ([]Document, error) {
+func (c *Client) Documents(ctx Context, caseType CaseType, caseId int, filterByDocType bool) ([]Document, error) {
```

Comment:
I wasn't initially clear what this variable did from the name. Could we change it to something like "onlyDrafts"? Or make it a string so downstream calls pass `sirius.TypeDraft` in?

Reply:
Oops initially I was trying to keep it flexible so other document types can be used but that's kind of pointless as only the draft doc type is added to the url in the method.. I'll change it to a string and leave that to be changed if it's needed in the future